src/harness/BenchmarkDotNet.Extensions/ExclusionFilter.cs

```diff
@@ -27,4 +29,41 @@ public bool Predicate(BenchmarkCase benchmarkCase)
             return !globFilter.Predicate(benchmarkCase);
         }
     }
+
+    class CategoryExclusionFilter : IFilter
+    {
+        private readonly List<(string userValue, Regex regex)> patterns;
+
+        public CategoryExclusionFilter(List<string> patterns)
+        {
+            if (patterns != null)
+            {
+                this.patterns = patterns.Select(pattern => (pattern, new Regex(WildcardToRegex(pattern), RegexOptions.IgnoreCase | RegexOptions.CultureInvariant))).ToList();
+            }
+            else
+            {
+                patterns = null;
+            }
+        }
+
+        public bool Predicate(BenchmarkCase benchmarkCase)
+        {
+            if(patterns == null)
+            {
+                return true;
+            }
+            foreach (var category in benchmarkCase.Descriptor.Categories)
+            {
+                if(patterns.Any(pattern => category.ToLowerInvariant().Equals(pattern.userValue.ToLowerInvariant()) || pattern.regex.IsMatch(category)))
```

Comment:
why `ToLowerInvariant` instead of `Equals(str, StringComparison.OrdinalIgnoreCase)`?

Reply:
No reason other than I thought of the other way first. I like your way better.